modules/end-user-guide/pages/url-parameter-for-the-ide.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: CONCEPT
+:description: URL parameter for the IDE
+:keywords: ide, editor, workspace-ide, workspace-editor
+:navtitle: URL parameter for the IDE
+:page-aliases: url-parameter-for-the-in-browser-ide.adoc
+
+[id="url-parameter-for-the-ide"]
+= URL parameter for the IDE
+
+If the URL for starting a new workspace doesn't contain a parameter specifying the integrated development environment (IDE), the workspace loads with the default IDE, which is Microsoft Visual Studio Code - Open Source.
```

Comment:
```suggestion
If the URL for starting a new workspace doesn't contain a parameter specifying the integrated development environment (IDE), the workspace loads with the default IDE: Microsoft Visual Studio Code - Open Source.
```

Reply:
Thank you. I'll apply this, but with one small change: instead of the colon, I'll use a comma:

```suggestion
If the URL for starting a new workspace doesn't contain a parameter specifying the integrated development environment (IDE), the workspace loads with the default IDE, Microsoft Visual Studio Code - Open Source.
```